CHANGES.md

```diff
@@ -11,6 +11,8 @@ x.x.x (unreleased)
   - Fix mouseup not firing if click & drag a region handle & release outside browser window (#2213)
   - Added new `showTooltip` param allowing disabling region `title` tooltip (#2213)
 - Nullify `onaudioprocess` on remove to not execute in background (#2218)
+- Add new `vertical` param enabling displaying waveforms vertically; currently not compatible
```

Comment:
the second part of the sentence can be used in the separate PR that adds regions plugin support. ps. are you planning on adding support in other plugins?

Reply:
I removed the regions plugin part of the PR as you requested and will submit it in a different PR, and updated CHANGES.md too. I should be able to add support for other plugins - I think it's mostly pretty straightforward. It's not immediately obvious to me what will have to be done for the svg stuff in the markers plugin but that might just be because I don't know svgs that well. I think I should be able to do it.